setup-master.yml

```diff
@@ -6,3 +6,15 @@
     - name: test
       debug:
         msg: hello world
+
+# Skripte aktualisieren
```

Comment:
Ich finde name: reicht als Kommentar

Reply:
Generell finde ich Kommentare gut, um die Struktur und Lesbarkeit von Code -speziell in einer noch nicht so bekannten Programmiersprache- zu verbessern.